sdk/python/feast/infra/offline_stores/file.py

```diff
@@ -93,7 +93,8 @@ def evaluate_historical_retrieval():
 
                 # Read offline parquet data in pyarrow format
                 filesystem, path = FileOfflineStore.__prepare_path(
-                    feature_view.input.path
+                    feature_view.input.path,
+                    feature_view.input.file_options.s3_endpoint_override,
```

Comment:
Won't this use the default proto string value (empty string) even if it's not set explicitly in the proto?

Reply:
@achals good point :) fixed